src/pages/FlowRun/TaskRunTable-Tile.vue

```diff
@@ -136,6 +139,35 @@ export default {
 <template>
   <v-card class="pa-2" tile>
     <CardTitle :title="tableTitle" icon="pi-task-run">
+      <v-select
+        slot="sort"
+        v-model="state"
+        deletable-chips
+        style="width: 180px;"
+        chips
+        solo
+        flat
+        dense
+        hide-details
+        :items="[
+          { header: 'Finished States' },
+          { divider: '...' },
+          'Failed',
+          'Success',
+          'Cancelled',
+          'Finished',
+          'Skipped',
+          'TimedOut',
+          { header: 'Scheduled States - To Re-run Task Run' },
```

Comment:
```suggestion
          { header: 'Scheduled States - To Re-run Task Run' },
```
I'm not sure this header makes sense for a sort label?

Reply:
oh my bad, I forgot to change that. I copied it from the set state modal. For the sort, I don't think we would need headers?